integrations-service/integrations/utils/integrations/spider.py

```diff
@@ -7,6 +11,12 @@
 from ...models import SpiderFetchOutput
 
 
+# Cache spider client instances
```

Comment:
Caching is not needed for initializing `SpiderLoader` instances as initializing an instance here doesn't require making any external connection